sgx/src/lib.rs

```diff
@@ -10,5 +10,7 @@
 #![deny(missing_docs)]
 
 pub mod types;
-#[cfg(feature = "std")]
+#[cfg(feature = "iocuddle")]
+pub mod iocuddle;
```

Comment:
```suggestion
pub mod ioctls;
```

Reply:
So you're saying we want to put `iocuddle` module behind a feature flag, correct?

Reply:
No. I was suggesting naming the module `ioctls` to avoid accidental collision with the `iocuddle` crate. IMHO there should be two features: `std` and `crypto`.

Reply:
Ah cool. Thanks for the clarification!

Reply:
Done!